src/steps/step_list.go

```diff
@@ -66,18 +66,17 @@ type WrapOptions struct {
 // Wrap wraps the list with steps that
 // change to the Git root directory or stash away open changes.
 func (stepList *StepList) Wrap(options WrapOptions, repo *git.ProdRepo) error {
-	previousBranch, err := repo.Silent.PreviouslyCheckedOutBranch()
-	if err != nil {
```

Comment:
This was causing bug #1549. Sometimes there isn't a previous branch. That's not a reason to cancel the entire command. Instead, it now doesn't add the `PreserveCheckoutHistoryStep` steps and continues.

Reply:
Nice fix. Did we add a test for this? I don't see one in this PR

Reply:
Yes: features/repo/unconfigured.feature

Reply:
That doesn't appear to me to be a test for not having a previously checked out branch

Reply:
Woops sorry, you are right! I'll think about how to add such a test.